### PR TITLE
feat: add OpenGithub - Github开源项目月刊精选

### LINKED ISSUE #15153: 抓取请求：open github 周刊
### Category

Programming

### Website URL

https://open.itc.cn/?tab=collection

### Website description

帮助用户发现最新鲜、最热门、最流行Github开源项目

### What content should be included?

已发布的周刊内容

### Additional description

_No response_

### This is not a duplicated rss request

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) and [pull requests](https://github.com/DIYgod/RSSHub/pulls) to ensure this rss proposal has not already been requested

### PR DESCRIPTION
**Pull request by ubbcou**
## Involved Issue / 该 PR 相关 Issue

Close #15153

## Example for the Proposed Route(s) / 路由地址示例

```routes
/itc/collection/0
/itc/collection/1
/itc/collection/2
/itc/collection/3
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

/itc/collection/0 - 精选文章
/itc/collection/1 - 专栏
/itc/collection/2 - 周刊
/itc/collection/3 - 月刊

### CODE REVIEW

**Comment by TonyRL** [comment]
Please remove all unnecessary files.

**Comment by ubbcou** [comment]
> Please remove all unnecessary files.

ok

File: lib/routes/itc/collection.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+async function handler(ctx) {
+    const url = 'https://open.itc.cn';
+
+    const colType = ctx.req.param('colType');
+
+    const result = {
+        title: '',
+        link: url,
+        description: RESULT_DESC_MAP[colType] ?? 'NULL',
+        item: [] as Data[],
```
Comment (TonyRL) [inline lib/routes/itc/collection.ts:47]: The type of returned object is `Data`. The type of `item` in the returned object is [DataItem](https://github.com/DIYgod/RSSHub/blob/a51cc049bf67c24216953449f1a2d500fe7f0367/lib/types.ts#L31).

File: lib/routes/itc/collection.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    result.item = [...$('.tab-pane > .row > .card')].map((item) => {
+        const date = $(item).find('.d-flex.mt-3.ms-sm-auto').text()?.split(':')?.[1];
+        const dataObject = date ? new Date(date) : undefined;
+
+        return {
+            title: $(item).find('a.btn-link').text(),
+            description: $(item).find('.d-sm-inline-block').text(),
+            link: $(item).find('a.btn-link').attr('href') || '',
+            pubDate: dataObject,
+            published: dataObject,
```
Comment (TonyRL) [inline lib/routes/itc/collection.ts:61]: This property doesn't exist.

**Pull request merged**